stories/browser/src/types.ts

```diff
@@ -299,11 +299,12 @@ export type TransactionConfirmationExternalState = {
 export type TransactionConfirmationMessage = InteractiveMessage<
     InteractiveMessageType.TransactionConfirmation,
     {
-        toAddress?: string;
-        onAddressPress?: () => void | Promise<void>;
-        recipientsCount?: number;
-        totalAmount: string | React.ReactElement<any, any>;
-        fees: string | React.ReactElement<any, any>;
+        recipient?: string | React.ReactElement<any, any>;
+        onRecipientPress?: () => void | Promise<void>;
+        amount: string | React.ReactElement<any, any>;
```

Comment:
I suggest to use `BigNumber` here instead of arbitrary React elements

Reply:
I've actually been thinking about it too.